src/index.js

```diff
@@ -77,13 +89,44 @@ export default ({
     ],
   }));
 
+  const remoteModuleBaseUrls = [];
   // support one-app-cli's "serve-module"
   // merge local with remote, with local taking preference
   oneAppDevCdn.get('/module-map.json', (req, response) => {
     settle([
       remoteModuleMapUrl
-        ? got(remoteModuleMapUrl).then(
-          r => JSON.parse(r.body),
+        ? got(remoteModuleMapUrl, {
+          agent: new ProxyAgent(),
```

Comment:
This works with `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` thanks to our good friend @crosscompile's [undocumented addition to the proxy-agent lib](https://github.com/TooTallNate/node-proxy-agent/pull/16)